source/modules/MEI.shared.xml

```diff
@@ -4922,58 +4877,32 @@
       </attDef>
     </attList>
   </elementSpec>
-  <elementSpec ident="composer" module="MEI.shared">
-    <desc>The name of the creator of the intellectual content of a musical work.</desc>
-    <classes>
-      <memberOf key="att.common"/>
-      <memberOf key="att.bibl"/>
-      <memberOf key="att.evidence"/>
-      <memberOf key="att.facsimile"/>
-      <memberOf key="att.lang"/>
-      <memberOf key="model.respLikePart"/>
-    </classes>
-    <content>
-      <rng:zeroOrMore>
-        <rng:choice>
-          <rng:text/>
-          <rng:ref name="model.textPhraseLike.limited"/>
-        </rng:choice>
-      </rng:zeroOrMore>
-    </content>
-  </elementSpec>
-  <elementSpec ident="contributor" module="MEI.shared">
-    <desc>Names of individuals, institutions, or organizations responsible for contributions to the
-      intellectual content of a work, where the specialized elements for authors, editors, etc. do
-      not suffice or do not apply.</desc>
+  <elementSpec ident="agent" module="MEI.shared">
+    <desc>The name of an individual or corporate body who or which is related to any other entity. 
+      The nature of the relationship should be expressed within the <att>role</att> attribute.</desc><!-- TODO: check if speaker,audience, distributor are needed-->
```

Comment:
Fix line length to match surrounding text.

Reply:
Do you mean the comment? It will be removed later anyway .